packages/orchestrator-ui-components/src/rtk/endpoints/streamMessages.ts

```diff
@@ -97,9 +97,12 @@ const streamMessagesApi = orchestratorApi.injectEndpoints({
                 const getDebounce = (delay: number) => {
                     return debounce(() => {
                         webSocket.close();
-                        // note: websocket.close doesnt trigger the onclose handler when losing
+                        // note: websocket.close doesn't trigger the onClose handler when losing
                         // internet connection so we call the cleanup event from here to be sure it's called
                         cleanUp();
+                        throw new Error(
```

Comment:
Doesn't this mean that when no wfoErrorMonitoring is implemented that these error become uncaught and will crash the application?

Reply:
Good point, will remove the error-throwing. Implementing it later